Symfony/CS/FixerFactory.php

```diff
@@ -76,6 +83,20 @@ public function getFixers()
     }
 
     /**
+     * Check if fixer exists.
+     *
+     * @param string $name The rule name of the fixer
+     *
+     * @return bool
+     */
+    public function hasFixer($name)
```

Comment:
This is quite confusing because it's still an array of Fixers.

What do you prefer? Description update? Rename the method to `hasRule`?